drivers/dotnet-installation.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+---
+# Installation
```

Comment:
This doesn't actually show how to install the driver. It is okay in the case of Nuget I suppose because the referenced site shows different ways to obtain the driver but as a beginner, I wouldn't know how to continue.

As there is very little content here and on the Getting Started page, I would suggest merging it.

Reply:
@Simran-B, I have added more details to the installation page. I was thinking of keeping Getting Started as an index page with links to others providing more details about each topic. Do you still think we should merge it?

Reply:
@Simran-B, let me know your final thoughts on this. Thanks